lib/almodovar.rb

```diff
@@ -16,14 +16,27 @@
 require 'almodovar/to_xml'
 
 module Almodovar
-
   class << self
+    DEFAULT_SEND_TIMEOUT = 30
+    DEFAULT_CONNECT_TIMEOUT = 30
+    DEFAULT_RECEIVE_TIMEOUT = 30
+
     def default_options
-      @default_options ||= {
-        :timeout => 120,
-        :connect_timeout => 30,
-        :user_agent => "Almodovar/#{Almodovar::VERSION}"
+      default = {
+        send_timeout: DEFAULT_SEND_TIMEOUT,
```

Comment:
@AlfonsoUceda I agree that this _might_ be useful in some cases, but:

1. We are _not_ using it (for us, for now).
2. None of the users of the gem have requested it.
3. If we provided it, I'd not expose the exact same interface that the http library we using, since it may change, and this is a contract we are going to define with the outer world.